JShellAPI/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": [
+    {
+      "name": "jshell-wrapper.image-name",
+      "type": "java.lang.String",
+      "description": "JShellWrapper image name injected from the top-level gradle build file."
+  }
+] }
```

Comment:
What is this ?

Reply:
It's used by your IDE so you get proper linting in the context of Spring @Alathreon

Reply:
I guess no problem here ? 😎